system/modules/core/classes/FrontendTemplate.php

```diff
@@ -250,7 +250,7 @@ protected function addToCache()
 			// Add the cache file header
 			$strHeader = sprintf
 			(
-				"<?php /* %s */ \$expire = %d; \$content = %s; \$type = %s; \$files = %s; \$assets = %s; ?>\n",
+				"<?php\n\n// %s\n\$expire = %d;\n\$content = %s;\n\$type = %s;\n\$files = %s;\n\$assets = %s;\n\n?>\n",
```

Comment:
Inserting the cache key as a PHP comment scares me a little bit because the URL is untrusted. There are ways to break out of single line comments like `?><?php`. I didn’t find a way to get these characters through the `Environment` class, but I’d prefer to play it safe and remove the comment completely from the cache file.